Jenkins Kubernetes Operator sub-project annoucement blog post (#4244)
jenkins-infra/jenkins.io: content/blog/2021/04/2021-04-15-jenkins-operator-sub-project.adoc

---
layout: post

> title: "Jenkins Operator becoming sub-project!"

Reviewer: ```suggestion
title: "Jenkins Operator becomes an official sub-project!"
```

tags:
- kubernetes
- operator

> - contributing

Reviewer: ```suggestion
- contributing
- announcement
```

authors:
- antoniaklja
- sylwiabrant
sig: cloud-native
opengraph:

>   image: https://virtuslab.com/wp-content/uploads/2019/04/jenkins_y6uwlf.jpg

Reviewer 2: not sure if we want an external image here? is there precedent? It wouldn't be good if this breaks in future.

Reviewer 2: I believe Oleg meant we could use the image, but not necessarily linking directly to it.

Author: fixed 👍

---

> We are happy to announce that Jenkins Operator officially link:https://www.jenkins.io/projects/jenkins-operator/[became Jenkins sub-project].

Reviewer: ```suggestion
We are happy to announce that Jenkins Operator officially link:/projects/jenkins-operator/[became an official Jenkins sub-project].
```

== What does it mean for the project?

image:/images/logos/kubernetes-operator/256.png["Jenkins Operator", role=left]

Becoming an official part of the Jenkins project was a major step towards better alignment with the overall Jenkins’ roadmap and more opportunities to increase adoption of the Jenkins Operator project.

> Finally, with a dedicated team at link:http://virtuslab.com/[VirtusLab] actively maintaining to the project we can engage with the wider community and participate in some of the link:/sigs/cloud-native/#meetings[Cloud-Native SIG meetings].

Reviewer: ```suggestion
Finally, with a dedicated team at link:http://virtuslab.com/[VirtusLab] actively maintaining the project we can engage with the wider community and participate in some of the link:/sigs/cloud-native/#meetings[Cloud-Native SIG meetings].
```

== Bridging the gap between Jenkins and Kubernetes

Running Jenkins in a cloud-native environment like Kubernetes is not a trivial task.
With Jenkins Operator project we want to enable the community to take full advantage of Kubernetes and public cloud capabilities by:

* native integration with public cloud services in areas of observability, storage and cloud security
* Kubernetes autoscaling and self-healing mechanism
* secure access to Jenkins instance
* declarative configuration using Kubernetes Custom Resources
* full lifecycle management, eventually transforming it to autopilot

== Take part in the journey toward global trends

*Start link:https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md[contributing] and play an important role in creating the automated Jenkins experience! Don’t hesitate - engage in the community.*
Join us in our work of creating the functionalities you see beneficial.
You are welcome to create issues and pull requests. We are actively resolving community issues and providing answers on a dedicated link:https://github.com/jenkinsci/kubernetes-operator#community[Slack channel].

> As the project has been initially developed and still actively maintained by link:http://virtuslab.com/[VirtusLab], we started link:https://groups.google.com/g/jenkinsci-dev/c/OA5nb_SAgh0/m/OoBS2o8nAwAJ:[discussion] towards open governance model to facilitate communication and collaboration.

Reviewer: ```suggestion
As the project has been initially developed and still actively maintained by link:http://virtuslab.com/[VirtusLab], we started link:https://groups.google.com/g/jenkinsci-dev/c/OA5nb_SAgh0/m/OoBS2o8nAwAJ:[discussion] towards an open governance model to facilitate communication and collaboration.
```

== Lead the way we are heading by providing us feedback

To celebrate the new possibilities that are being opened for our project, we would like to invite you to take part in a short survey that will help put us on the right track.
If you have been using Jenkins Operator or running Jenkins in any other environment, please take a moment to fill out our quick survey.
*We will choose at least three of the most informative answers and send you an awesome Jenkins Operator T-shirt with our cute Gopher butler.*
The survey can be found link:https://docs.google.com/forms/d/1doIkgnm3_WbjtlwWSU4sOoiI7QoneHlYIjXEJOVMrfQ/edit?usp=sharing[here]. Remember, the most sincere answers are the best.

image:/images/post-images/2021-04-jenkins-operator/tshirt-logo.jpg[T-shirt, width=400px]

> And if you still haven’t used Jenkins Kubernetes Operator, you’re more than welcome to do so. Give it a shot and discover a new kind of simplicity of running Jenkins.

Reviewer: ```suggestion
And if you still haven’t used Jenkins Kubernetes Operator, you’re more than welcome to do so.
Give it a shot and discover a new kind of simplicity of running Jenkins.
link:/projects/jenkins-operator/#getting-started[How to get started].
```
